src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -573,7 +573,8 @@ def fn(
                     zerofill_results=zerofill_results,
                     comparison_delta=comparison_delta,
                     dataset=dataset,
-                    transform_alias_to_input_format=transform_alias_to_input_format,
+                    # use_rpc should always transform_alias_to_input_format
+                    transform_alias_to_input_format=transform_alias_to_input_format or use_rpc,
```

Comment:
Thoughts on moving this up to where `transform_alias_to_input_format` is initialized?

so `transform_alias_to_input_format = use_rpc or request.GET.get(...)`